app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,34 @@
+class PagesController < ApplicationController
+  def home
+  end
+    
+  def index 
+    @journal_entry = JournalEntry.all
+  end
+
+  def new_entry_form
```

Comment:
```suggestion
  def new
```

Reply:
if i later want to create tasks and goals wont i be easyer with this naming ?

Reply:
if you want to create tasks then you need to create it from the task controller and not from pages controller